cmd/main.go

```diff
@@ -1,53 +1,60 @@
 package main
 
 import (
-	"TicTacGo/internal/board"
-	"fmt"
+	"fmt" // first stdlib imports
+
+	// after empty line 3rd-party imports
+
+	// always use full import paths
+	// imternal package is needed only for packages that is explicitly forbidden to use from outside
+	"github.com/Kwbmm/TicTacGo/board" // after empty line local module imports
```

Comment:
According to [this](https://github.com/golang-standards/project-layout) then I should move board package inside `pkg` folder.

Reply:
For all years of Go programming I've seen pkg folder really rarely.
In my personal opinion, if you can get rid of something (within a reasonable), you should do it.